### DISCUSSION TITLE
When was config GUI added?

### DISCUSSION THREAD

**Discussion by Kadooment** [maruohon/minihud#239] [Q&A] [answered]
I'm playing on 1.12.2 and I'm unable to open the config menu in game like I'm used to with a newer version. I'm wondering if that's because it doesn't exist? Pressing H+C just toggles sub-chunk info in the mini-f3 portion, but I'd like to enable other overlay features features (structure bounding boxes, slime chunks, etc.) but can't find a way to without closing the game and messing with the config file.

**Comment by maruohon** [comment, marked as answer]
Which mod version are you using? The old **0.13.1** Forge version doesn't have the in-game config screen yet, it only has the simple config menu via the Forge mod list. And it also only has the Info Lines feature, I think?

There are newer, malilib-based versions of the mod here: https://masa.dy.fi/mcmods/client_mods/

The Forge versions there are currently pretty old, from December 2019. (Even if the build date is newer, the code is from the LiteLoader version from December 2019, there might just be some tiny fix or something made more recently)

The LiteLoader versions are the latest development code. Even those don't currently still have all the features yet that were recently added to the 1.18 versions. I still need to backport those things (like the biome border overlay).

**Comment by Kadooment** [comment]
Ah yeah that would explain it. I was using 0.13.1, though I'm pretty sure that one is malilib-based because I had to install that too. I was using it with pixelmon, but since they just dropped a 1.16.5 update yesterday this probably won't be a problem for me moving forward. Thanks for the response!

**Reply by maruohon** [reply]
No 0.13.1 doesn't use malilib. If I'm to believe the malilib CurseForge page (as I can't remember anymore off the top of my head), MiniHUD 0.17.0 and later require malilib. So apparently there were a number of later releases still that didn't need it... 🤔

**Reply by Kadooment** [reply]
Interesting, could've sworn it said it was required when I tried launching without it, but that must've been a different game version I was using before playing on 1.12.2
